src/LaunchDarkly.EventSource/StateChangedEventArgs.cs

```diff
@@ -16,13 +17,23 @@ public class StateChangedEventArgs : EventArgs
         /// </value>
         public ReadyState ReadyState { get; }
 
+        /// <summary>
+        /// Get the response headers. Only populated for <see cref="ReadyState.Open"/>.
+        /// <value>
+        /// A collection of header values when the ReadyState is Open, or null.
+        /// </value>
+        /// </summary>
+        public IEnumerable<KeyValuePair<string,IEnumerable<string>>> Headers { get; }
+
         /// <summary>
         /// Initializes a new instance of the <see cref="StateChangedEventArgs"/> class.
         /// </summary>
         /// <param name="readyState">One of the <see cref="EventSource.ReadyState"/> values, which represents the state of the EventSource connection.</param>
-        public StateChangedEventArgs(ReadyState readyState)
+        /// <param name="headers">Response headers when the <see cref="StateChangedEventArgs.ReadyState"/> is <see cref="ReadyState.Open"/>. Otherwise null.</param>
+        public StateChangedEventArgs(ReadyState readyState, IEnumerable<KeyValuePair<string,IEnumerable<string>>> headers = null)
```

Comment:
This constructor is public, but isn't something someone would really need to use in normal operation.

Likely it should have been internal.